integreat_cms/cms/models/languages/language_tree_node.py

```diff
@@ -93,6 +100,25 @@ def text_direction(self):
         """
         return self.language.text_direction
 
+    @cached_property
+    def mt_provider(self):
+        """
+        Return the name of the machine translation provider it it exists,
+        or empty string otherwise
+
+        :return: Name of the MT provider for the language
+        :rtype: str
+        """
+        deepl_config = apps.get_app_config("deepl_api")
+        if (
+            self.slug == settings.SUMM_AI_EASY_GERMAN_LANGUAGE_SLUG
+            and settings.SUMM_AI_ENABLED
```

Comment:
```suggestion
            and settings.SUMM_AI_ENABLED
            and self.region.summ_ai_enabled
```
We also need to check here whether SUMM.AI is enabled in the current region